cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -61,4 +61,23 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 30
+          timeoutSeconds: 5
+      - name: healthz
+        image: gcr.io/google_containers/exechealthz:1.0
+        resources:
+          limits:
+            cpu: 10m
+            memory: 20Mi
+        args:
+        - -cmd=nslookup kubernetes.default.svc.{{ pillar['dns_domain'] }} localhost >/dev/null
```

Comment:
I want this to be an absoluet lookup

On Fri, Jul 10, 2015 at 9:05 AM, Abhi Shah <notifications@github.com> wrote:

> In cluster/addons/dns/skydns-rc.yaml.in
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11004#discussion_r34369250>
> :
>
> > @@ -61,4 +61,23 @@ spec:
> >          - containerPort: 53
> >            name: dns-tcp
> >            protocol: TCP
> > +        livenessProbe:
> > +          httpGet:
> > +            path: /healthz
> > +            port: 8080
> > +            scheme: HTTP
> > +          initialDelaySeconds: 30
> > +          timeoutSeconds: 5
> > +      - name: healthz
> > +        image: gcr.io/google_containers/exechealthz:1.0
> > +        resources:
> > +          limits:
> > +            cpu: 10m
> > +            memory: 20Mi
> > +        args:
> > +        - -cmd=nslookup kubernetes.default.svc.{{ pillar['dns_domain'] }} localhost >/dev/null
>
> you should not need to say pillar['dns_domain']
> just kubernetes.default.svc should be enough
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/11004/files#r34369250>
> .
>